### PR TITLE
Added stripe payment support

### PR DESCRIPTION
**Pull request by Daniel-Ozori**

### CODE REVIEW

File: stores/Payment.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
+import env from 'rcompat/env';
+import { primary } from '@primate/types';
+
+export const actions = ({connection: db}) => {
+    return{
+        async create(data){
+
+			let { userId, status, subscriptionInterval, stripeSubscriptionId, productId, renewalDate, cancelAtPeriodEnd  } = data;
```
Comment (ralyodio) [inline stores/Payment.js:8]: prettier

File: stores/Product.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        async getByStripeProductId(id){
+            const query = `SELECT * FROM products WHERE stripeProductId = $id`;
+
+            try{
+
+                const product = await db.query(query, {
+					id
+				});
+
+				return product.pop().pop();
```
Comment (ralyodio) [inline stores/Product.js:28]: prettier

**Review by ralyodio — CHANGES_REQUESTED** [review]
just a few changes. please run `prettier`

**Comment by Daniel-Ozori** [comment]
Changes made

**Pull request merged**